build.sbt

```diff
@@ -155,6 +156,32 @@ lazy val semanticdbScalacPlugin = project
   )
   .dependsOn(semanticdbScalacCore)
 
+lazy val semanticdbJavacPlugin = project
+  .in(file("semanticdb/javac"))
+  .settings(
+    moduleName := "semanticdb-javac",
+    description := "Javac compiler plugin that generates SemanticDB on compile",
+    publishableSettings,
+    mergeSettings,
+    fullCrossVersionSettings,
```

Comment:
This project doesn't have to be `CrossVersion.full`. `semanticdbScalac` has to, because it depends on `scala-compiler.jar`, but only because of that.